### DISCUSSION TITLE
Is there any reason why there is no way to install it via cocoapods?

### DISCUSSION THREAD

**Discussion by OmarJalil** [gonzalezreal/swift-markdown-ui#225] [General]
I think cocoapods is still a really big community

**Comment by gonzalezreal** [comment]
Hi @OmarJalil,

I develop this library in my spare time, and maintaining a pod spec is time-consuming. So, I prefer to focus on improving the library itself.

I am not a Cocoapods expert, but keeping pod specs outside the library's repo is possible. If someone were to maintain a pod spec for MarkdownUI, I would happily link it in the README.

Thanks for understanding.
